internal/resources/kubernetessecret/scope/constants.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package scope
+
+const (
+	AttachedValue            = "attached"
+	ScopeKey                 = "scope"
+	ClusterKey               = "cluster"
+	ClusterGroupKey          = "cluster_group"
+	ManagementClusterNameKey = "management_cluster_name"
+	ProvisionerNameKey       = "provisioner_name"
+	ClusterNameKey           = "cluster_name"
+)
+
+const (
+	UnknownScope Scope = iota
+	ClusterScope
+	ClusterGroupScope
```

Comment:
`namespace` scope and `cluster group namespace` scopes

Reply:
@Shubbhang351 can you check with the team on this? It should be at par with the scope being added to the Tanzu CLI plugin.

Reply:
As discussed keep it `cluster` and `cluster_group`